doc/server_config.md

```diff
@@ -31,6 +31,8 @@ Make sure you have the following packages installed:
   * xml2 **(Version 1.0 or higher)**
   * jaod (Currently, a Github repository only: http://github.com/ropenscilabs/jaod. Install with  devtools.)
   * rbace (Currently, a Github repository only: http://github.com/ropenscilabs/rbace. Install with devtools.)
+  * ropenaire (For VIPER. Currently, a Github repository only: https://github.com/njahn82/ropenaire. Install with devtools.)
```

Comment:
This should be changed to Scott's repository for now (before it is merged back into Najko's).